src/core/ipns/routing/utils.js

```diff
@@ -1,8 +1,7 @@
 'use strict'
 
-const base32 = require('base32.js')
+const multibase = require('multibase')
 
 module.exports.encodeBase32 = (buf) => {
-  const enc = new base32.Encoder()
-  return enc.write(buf).finalize()
+  return multibase.encode('base32', buf)
```

Comment:
If you only want the base32 encoding of `buf` you'll need to slice off the multibase codec from the front, or require and use the base32 module directly https://github.com/multiformats/js-multibase/blob/master/src/base32.js.